app/javascript/components/dashboard/routes/AudienceRoute.tsx

```
@@ -0,0 +1,52 @@
+
+  if (error) {
+    return <ErrorFallback error={new Error(error)} />;
+  }
+
+  if (isLoadingAudience && !audienceData) {
+    return <PageLoading message="Loading audience data..." />;
+  }
+
+  // For now, redirect to the existing audience page
+  // TODO: Create a React-based audience component
+  useEffect(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)